src/index.html

```diff
@@ -5,8 +5,433 @@
+        </nav>
+      </div>
+    </header>
+
+    <main>
+      <section class="catalog">
+        <catalog class="card catalog__item" data-qa="card">
+          <img
+            src="images/imac.jpeg"
+            alt="Item img"
+            class="card__img"
+          >
+
+          <h2 class="card__desc">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <p class="card__code">
+            Product code: 195434
+          </p>
+
+          <div class="card__stars-rate">
+            <div class="stars stars--4">
+              <div class="stars__star-rate"></div>
+              <div class="stars__star-rate"></div>
+              <div class="stars__star-rate"></div>
+              <div class="stars__star-rate"></div>
+              <div class="stars__star-rate"></div>
+            </div>
+
+            <p class="card__rewiews-count">
+              Reviews: 5
+            </p>
+          </div>
+
+          <div class="card__price">
+            <p class="card__price-text">
+              Price:
+            </p>
+
+            <p class="card__price-amount">
+              $2,199
+            </p>
+          </div>
+          <a
+            href="#"
+            class="card__button"
+            data-qa="card-hover"
+          >
+            Buy
+          </a>
+        </catalog>
+
+        <catalog class="card catalog__item">
+          <img
+            src="images/imac.jpeg"
+            alt="Item img"
+            class="card__img"
+          >
+
+          <h2 class="card__desc">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <p class="card__code">
+            Product code: 195434
+          </p>
+
+          <div class="card__stars-rate">
+            <div class="stars stars--4">
+              <div class="stars__star-rate"></div>
+              <div class="stars__star-rate"></div>
+              <div class="stars__star-rate"></div>
+              <div class="stars__star-rate"></div>
+              <div class="stars__star-rate"></div>
+            </div>
+
+            <p class="card__rewiews-count">
+              Reviews: 5
+            </p>
+          </div>
+
+          <div class="card__price">
+            <p class="card__price-text">
+              Price:
+            </p>
+
+            <p class="card__price-amount">
+              $2,199
+            </p>
+          </div>
+          <a
+            href="#"
+            class="card__button"
+          >
+            Buy
+          </a>
+        </catalog>
+
+        <catalog class="card catalog__item">
+          <img
+            src="images/imac.jpeg"
+            alt="Item img"
+            class="card__img"
+          >
+
+          <h2 class="card__desc">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <p class="card__code">
+            Product code: 195434
+          </p>
+
+          <div class="card__stars-rate">
```

Comment:
```suggestion
          <div class="card__stars--rate">
```
If it's a modifier you should use double dash. Check typical BEM mistakes [here](https://mate-academy.github.io/fe-program/css/typical-bem-mistakes.html)